test/unit/behaviors.spec.js

```diff
@@ -410,509 +508,600 @@ describe('Behaviors', function() {
+        fooView.delegateEvents();
+        expect(fooBehavior.ui.foo.click.bind(fooView.ui.bar)).to.not.throw(Error);
+        expect(fooView.ui.bar.click.bind(fooView.ui.bar)).to.not.throw(Error);
       });
 
       it('should set the behavior UI element', function() {
-        expect(this.onRenderStub).to.have.been.calledOnce;
+        expect(onRenderStub).to.have.been.calledOnce;
       });
 
       it('should make the view\'s ui hash available to callbacks', function() {
-        expect(this.fooBehavior.testViewUI.bind(this.fooBehavior)).to.not.throw(Error);
+        expect(fooBehavior.testViewUI.bind(fooBehavior)).to.not.throw(Error);
       });
 
       it('should make the behavior\'s ui hash available to callbacks', function() {
-        expect(this.fooBehavior.testBehaviorUI.bind(this.fooBehavior)).to.not.throw(Error);
+        expect(fooBehavior.testBehaviorUI.bind(fooBehavior)).to.not.throw(Error);
       });
 
       describe('the $el', function() {
         beforeEach(function() {
-          this.view.$el.find('.foo').click();
-          this.view.$el.find('.bar').click();
+          fooView.$el.find('.foo').click();
+          fooView.$el.find('.bar').click();
         });
 
         it('should handle behavior ui click event', function() {
-          expect(this.onFooClickStub).to.have.been.calledOnce.and.calledOn(this.fooBehavior);
+          expect(onFooClickStub).to.have.been.calledOnce.and.calledOn(fooBehavior);
         });
 
         it('should handle view ui click event', function() {
-          expect(this.onBarClickStub).to.have.been.calledOnce.and.calledOn(this.fooBehavior);
+          expect(onBarClickStub).to.have.been.calledOnce.and.calledOn(fooBehavior);
         });
 
         it('has a getUI method which returns the selector', function() {
-          expect(this.fooBehavior.getUI('foo')).to.have.length(1);
+          expect(fooBehavior.getUI('foo')).to.have.length(1);
         });
       });
 
       describe('the el', function() {
         beforeEach(function() {
-          $(this.view.el).find('.foo').click();
-          $(this.view.el).find('.bar').click();
+          $(fooView.el).find('.foo').click();
+          $(fooView.el).find('.bar').click();
         });
 
         it('should handle behavior ui click event', function() {
-          expect(this.onFooClickStub).to.have.been.calledOnce.and.calledOn(this.fooBehavior);
+          expect(onFooClickStub).to.have.been.calledOnce.and.calledOn(fooBehavior);
         });
 
         it('should handle view ui click event', function() {
-          expect(this.onBarClickStub).to.have.been.calledOnce.and.calledOn(this.fooBehavior);
+          expect(onBarClickStub).to.have.been.calledOnce.and.calledOn(fooBehavior);
         });
       });
     });
 
     describe('within a layout', function() {
       beforeEach(function() {
         this.setFixtures('<div id="layout"></div>');
-        this.ItemView = Marionette.View.extend({
+        let BarView = View.extend({
```

Comment:
This might just be personal preference, and I have seen arguments either way, but I do think the way Marionette is currently doing things is that if it can be a `const` it should be.  And this and `barView` below (I think) should be `const`.

There are other examples of using `let` when it could be `const`  I don't know how important, if at all, it is that it is changed.